### PR TITLE
Adjust provisioning profiles for widgets

### PR DESCRIPTION
**Pull request by brunobar79**
Fixes RNBW-####

## What changed (plus any additional context for devs)
Testflight builds currently failing on CI because we never created provisioning profiles for the appstore for the PriceWidget and SelectTokenIntent targets.  I've created those and set it on xcode config.

## PoW (screenshots / screen recordings)
None

## Dev checklist for QA: what to test
Both e2e and testflight builds should pass

## Final checklist
[x] Assigned individual reviewers?
[x] Added labels?

### CODE REVIEW

**Review by osdnk — APPROVED** [review]
❤️

File: ios/Rainbow.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1642,7 +1643,7 @@
 				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
 				CODE_SIGN_ENTITLEMENTS = PriceWidgetExtension.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Distribution";
-				CODE_SIGN_STYLE = Automatic;
```
Comment (osdnk) [inline ios/Rainbow.xcodeproj/project.pbxproj:1645]: Do we need this?

Comment (osdnk) [reply ios/Rainbow.xcodeproj/project.pbxproj:1645]: Hm looks like we neeed

**Review by estebanmino — APPROVED** [review]
🍰

**Pull request merged**